.github/workflows/sync-comp.yml

```diff
@@ -0,0 +1,62 @@
+name: CaC to OSCAL Component Definition
+on:
+  workflow_dispatch:
+   inputs:
+      cac-reference:
+        description: 'Compliance as Code git reference'
```

Comment:
Nit: Below we are using `CaC`. Just for consistence we could use `ComplianceAsCode` or `CaC` since these options are more commonly used.